app/models/flood_risk_engine/organisation.rb

```diff
@@ -2,6 +2,7 @@ module FloodRiskEngine
   class Organisation < ActiveRecord::Base
     belongs_to :contact
     has_one :enrollment, dependent: :restrict_with_exception
+    has_many :partners
```

Comment:
I agree with @autotelik this is a bit misleading. In WEX we subclassed all org types using STI but in flood this is the only piece of code that is org-class specific like this, so that approach a bit overkill. I can't think of a solution but we could add a comment to make it clear it applies only when the Organisation is a partnership. There are conditions on a has_many (see second answer [here](http://stackoverflow.com/questions/9556153/is-it-possible-to-create-a-conditional-association-in-model)) but that's more about scoping I suppose, and isn't going to make that relationship disappear for non partnership Organisations.